test/CMakeLists.txt

```diff
@@ -149,6 +149,8 @@ addLinkAndDiscoverTest(MultiColumnJoinTest engine)
 
 addLinkAndDiscoverTest(IdTableTest util)
 
+addLinkAndDiscoverTest(GrbMatrixTest engine)
```

Comment:
This test also has to be linked against `graphblas`, that's probably why the `measure coverage` build currently doesn't work.